src_bonus/Makefile

```diff
@@ -0,0 +1,91 @@
+######################################
+#              COMMANDS              #
+######################################
+CXX         :=   ${shell which c++}
+LINK        :=   ${shell which clang++}
+MKDIR       :=   ${shell which mkdir} -p
+RM          :=   ${shell which rm} -rf
+CMAKE		:=	${shell which cmake}
+
+######################################
+#             EXECUTABLE             #
+######################################
+NAME := bot
+
+#######################################
+#             DIRECTORIES             #
+#######################################
+OBJ_DIR     :=   obj
+SRC_DIR		:=	.
+PRIV_DIR	:=	../private/
+INC_DIR		:=	../include/
+
+#######################################
+#              LIBRARIES              #
+#######################################
+
+IRC_A := ../libircserv.a
+
+######################################
+#            SOURCE FILES            #
+######################################
+SRC := \
+	${addsuffix .cpp, \
+	 \
+		${addprefix Bot/, \
+			core \
+			methods \
+		} \
+		main \
+	}
+
+#######################################
+#                FLAGS                #
+#######################################
+CXXFLAGS    = \
+	-c \
+	-Wall \
+	-Wextra \
+	-Werror \
+	-Wshadow \
+	-pedantic \
+	-std=c++14 \
```

Comment:
We must code in c++98 standard, thus, you must change this compilation flag to `-std=c++98`.